erpnext_documentation/www/docs/user/manual/en/stock/sales-return.md

```diff
@@ -39,6 +39,17 @@ Before creating and using a Sales Return, it is advised that you create the foll
 
 If Perpetual Inventory is enabled, the system will also post accounting entry against warehouse account to sync warehouse account balance with stock balance as per Stock Ledger.
 
-## 3. Related Topics
+## 3. Impact on Stock Return via Delivery Note
+On Creating a Sales Return against a Delivery Note:
+
+* The **Returned Quantity** in the original Delivery Note along with any Sales Order linked to it, is updated.
+
+* The original Delivery Note's status is changed to **Return Issued** if 100% returned.
+  ![Return Issued](/docs/assets/img/stock/sales-return-issue.png)
+
+* The Returned Amount % is visible via the List View as well
+  ![Purchase Receipt List View](/docs/assets/img/stock/delivery-note-list.png)
```

Comment:
shouldn't this be.. ![Delivery Note List View](/docs/assets/img/stock/delivery-note-list.png) ?